### PR TITLE
Bump to v1.23.4

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
could you update changelog.txt too please @rhatdan ?

**Comment by TomSweeneyRedHat** [comment]
Otherwise LGTM

**Comment by flouthoc** [comment]
Commit needs `[NO NEW TESTS NEEDED]` but i feel we can ignore it at this point, PR is just a version bump.

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,9 @@
+- Changelog for v1.23.4 (2022-04-07)
+
+    chroot: run gofmt on run.go
+    vendor: bump c/image to v5.16.0-rhel/8b06d33
+    Use race-free c/storage by bumping it to 1.36.3
```
Comment (TomSweeneyRedHat) [inline changelog.txt:5]: Nit picking, this file has leading asterisks....

**Comment by flouthoc** [comment]
@rhatdan @TomSweeneyRedHat This is merged.

**Comment by flouthoc** [comment]
Just a new github cut is needed for `v1.23.4`

**Pull request merged**